src/run_solver.jl

```diff
@@ -0,0 +1,86 @@
+export solve_problems
+
+"""
+    solve_problems(solver, problems :: Any; kwargs...)
+
+Apply a solver to a set of problems.
+
+#### Arguments
+* `solver`: the function name of a solver
+* `problems`: the set of problems to pass to the solver, as an iterable of
+  `AbstractNLPModel`.  It is recommended to use a generator expression (necessary for
+  CUTEst problems).
+
+#### Keyword arguments
+* `solver_logger::AbstractLogger`: logger wrapping the solver call. (default: `NullLogger`).
+* `skipif::Function`: function to be applied to a problem and return whether to skip it
+  (default: `x->false`)
+* `prune`: do not include skipped problems in the final statistics (default: `true`)
+* any other keyword argument to be passed to the solver
+
+#### Return value
+* a `DataFrame` where each row is a problem, minus the skipped ones if `prune` is true.
+"""
+function solve_problems(solver, problems;
+                        solver_logger :: AbstractLogger=NullLogger(),
+                        reset_problem :: Bool = true,
+                        skipif :: Function=x->false,
+                        colstats :: Vector{Symbol} = [:name, :nvar, :ncon, :status, :elapsed_time, :objective, :dual_feas, :primal_feas],
+                        info_hdr_override :: Dict{Symbol,String} = Dict{Symbol,String}(),
+                        prune :: Bool=true, kwargs...)
```

Comment:
`reset_problem`, `colstats` and `info_hdr_override` aren't described in the docstring.